lib/MatterbridgeManager.php

```diff
@@ -493,6 +503,7 @@ private function checkBridgeProcess(Room $room, array $bridge, bool $relaunch =
 					$killed = $this->killPid($bridge['pid']);
 					if ($killed) {
 						$pid = 0;
+						$this->sendSystemBridgeStop($room);
```

Comment:
This is not what I had in mind. I would just do it when a bridge config is done. Not based on the real process.

`{actor} set up Matterbridge to synchronize this conversation with other chats.`
`{actor} updated the Matterbridge configuration.`
`{actor} removed the Matterbridge configuration.`

Reply:
> I would just do it when a bridge config is done

Not sure I understand what you mean. That's more or less what's done here. When the config is generated and the bridge is started or stopped, there is a system message. Do you mean you would like to trigger a system message each time the config file is edited?

I think it makes more sense to notify only when the bridge is started and stopped because the config can be edited many times without starting the bridge. The important information for the user is to know when messages are forwarded or not. Config change events seem less pertinent in a user point of view.

Reply:
> Do you mean you would like to trigger a system message each time the config file is edited?

Yeah and only then. Not otherwise you will create a chat message in each chat when the server is rebooted e.g.
I want users to be aware when their chat messages are leaked to other chats, they don't need to know if this services failed right now or not. And other moderators can then check what service was configured when "User X changed the matterbridge configuration" is posted into the chat. Logging who changed something crucial like this is the minimum. It's like we log when a chat is allowed/disallowing guests.

Reply:
Ok. Tiny remaining problem is that currently it will produce many system messages because the config is saved each time a part is added/removed. I'll prevent that as the save button is enough.